uniauth/views.py

```diff
@@ -748,3 +754,15 @@ def get_context_data(self, **kwargs):
         context.update(_get_global_context(self.request))
         return context
 
+
+def get_jwt_tokens_from_session(request):
+    if request.method == "POST":
+        refresh = request.session.pop("jwt-refresh", None)
+        access = request.session.pop("jwt-access", None)
+        if refresh is None or access is None:
+            return JsonResponse({}, status=status.HTTP_404_NOT_FOUND)
+        else:
+            return JsonResponse({
+                "refresh": refresh,
+                "access": access
+            }, status=status.HTTP_200_OK)
```

Comment:
❓Could you explain again why this needs to be a POST request? Since we're authenticating via a session cookies now, a GET seems like it should work, and it would side-step needing to pass a CSRF token too.

Reply:
Sorry, this is just an oversight on my part. You are indeed correct that it can be replaced with a GET request and avoid the CSRF token. I will fix that immediately. Great catch!